components/collective-page/sections/Projects.tsx

```diff
@@ -0,0 +1,181 @@
+  const oneCardScrollDistance = CONTRIBUTE_CARD_WIDTH + CONTRIBUTE_CARD_PADDING_X[0] * 2;
+  if (width <= oneCardScrollDistance * 2) {
+    return oneCardScrollDistance;
+  } else if (width <= oneCardScrollDistance * 4) {
+    return oneCardScrollDistance * 2;
+  } else {
+    return oneCardScrollDistance * 3;
+  }
+}
+
+export default function Projects(props: ProjectsProps) {
+  const { collective, isAdmin } = props;
+  const intl = useIntl();
+  const hasProjectsSection = (props.projects.length >= 0 && !collective.isActive) || isAdmin;
+
+  const [searchTerm, setSearchTerm] = React.useState('');
+  const deboucedSearchTerm = useDebounced(searchTerm, 1000);
+
+  const query = useQuery(
+    gql`
+      query ProjectsSection($slug: String, $searchTerm: String) {
+        account(slug: $slug) {
+          ... on AccountWithProjects {
+            projects(offset: 0, limit: 10, term: $searchTerm) {
+              totalCount
+              nodes {
+                ...ProjectSectionCardFields
+              }
+            }
+          }
+        }
+      }
+
+      ${ProjectSectionCardFields}
+    `,
+    {
+      context: API_V2_CONTEXT,
+      variables: {
+        slug: props.collective.slug,
+        searchTerm: deboucedSearchTerm,
+      },
+      skip: isEmpty(deboucedSearchTerm),
+    },
+  );
+
+  const refetch = query.refetch;
+  React.useEffect(() => {
+    refetch();
+  }, [deboucedSearchTerm, refetch]);
+
+  const searchProjects = React.useMemo(
+    () => (query.data?.account?.projects?.nodes || []).filter(p => isAdmin || !p.isArchived),
+    [isAdmin, query.data?.account?.projects?.nodes],
+  );
+  const collectiveProjects = React.useMemo(
+    () => props.projects.filter(p => isAdmin || !p.isArchived),
+    [isAdmin, props.projects],
+  );
+
+  const displayedProjects = isEmpty(deboucedSearchTerm) ? collectiveProjects : searchProjects;
+  if (!hasProjectsSection) {
+    return null;
+  }
+
+  return (
+    <Box pt={[4, 5]} data-cy="Projects">
+      <ContainerSectionContent>
+        <SectionTitle>
+          <FormattedMessage id="Projects" defaultMessage="Projects" />
+        </SectionTitle>
+        <P color="black.700" mb={4}>
+          {isAdmin ? (
+            <FormattedMessage
+              id="CollectivePage.SectionProjects.AdminDescription"
+              defaultMessage="Manage finances for a project or initiative separate from your collective budget."
+            />
+          ) : (
+            <FormattedMessage
+              id="CollectivePage.SectionProjects.Description"
+              defaultMessage="Support the following initiatives from {collectiveName}."
+              values={{ collectiveName: collective.name }}
+            />
+          )}
+        </P>
+        <Input
+          placeholder={intl.formatMessage({ defaultMessage: 'Search projects...', id: 'Dw9Bae' })}
+          type="search"
+          value={searchTerm}
+          onChange={e => setSearchTerm(e.target.value)}
+        />
```

Comment:
Maybe display this one if there are more than 10 projects total? Collectives with only 3 projects won't need a search bar there.

Reply:
Marking this one as unresolved as it doesn't seem addressed:

![image](https://github.com/user-attachments/assets/b84068bd-a956-4971-b8c0-c01d323f2337)